server.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const candidates_routes = require('./routes/candidates_routes.js')
+const ping_pong_routes = require('./routes/ping_pong_routes')
+
+const app = express();
+
+
+
+app.listen(12345, () => console.log('My App is listening on port 12345.'));
```

Comment:
The `candidates_routes` and `ping_pong_routes` are being required, but they are not used? 
We should integrate the routers into the main express application here.